yoti_python_sdk/tests/test_activity_details.py

```diff
@@ -119,8 +136,9 @@ def test_try_parse_age_verified_field_age_under():
 def test_try_parse_age_verified_field_non_bool_value_not_parsed():
     activity_details = ActivityDetails(successful_receipt())
     create_age_verified_field(activity_details, True, "18".encode(), 18)
-
+    sys.stdout = open(os.devnull, 'w')  # disable print
```

Comment:
What is the reason for disabling and re-enabling stdout?

Reply:
It just logs the error about parsing the age verified value when it's a non-bool value, so this is just disabling that for the line where we expect it to happen.